File: Python/Ex-40_Teams.py
# 
# Ex-40_Teams.py
# Created by Sebastian Jusca on 17/04/2023.
#

year = int(input('Enter your year: '))

if year % 4 != 0:
	print('The year {} is not a leap year.'.format(year))
elif year % 100 == 0 and year % 400 != 0:
	print('The year {} is not a leap year.'.format(year))
else:
	print('The year {} is a leap year'.format(year))

File: C/Ex-40_Teams.c
//
//  Ex-40_Teams.c
//  Ex 40 Explained on Microsoft Teams
//  30042023
//
//  Created by Sebastian Jusca on 29/03/2023.
//

#include <stdio.h>

int main()
{
    int year;
    
    printf("Enter your year: ");
    scanf("%d",& year);
    
    if(year % 4 != 0){
        printf("The year %d is not a leap year.\n", year);
    }else if(year % 100 == 0 && year % 400 != 0){
        printf("The year %d is not a leap year.\n", year);
    }else{
        printf("The year %d is a leap year.\n", year);
    }
    
    return 0;
}
